Update README conda package installation (#545)
RemDelaporteMathurin/FESTIM: docs/source/getting_started.rst

Reviewer: In this page, the Test section needs to go as it's not relevant for a end-user install

FESTIM requires FEniCS to run.

> The FEniCS project provides a prebuilt Anaconda python package (Linux and MacOS only) with FEniCS and its dependencies (python3, UFL, DOLFIN, numpy, sympy...)  already installed. 

Reviewer: ```suggestion
FEniCS can be installed with Anaconda on MacOs and Linux. 
```

> Anaconda can also be used in Windows using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) with your favourite linux distro. 

Reviewer: ```suggestion
In order to use the Anaconda distribution on Windows, use the [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install). 
```

Reviewer: Also, need to fix syntax for url link

> First [install Anaconda](https://docs.continuum.io/anaconda/install) then run the following commands 

Reviewer: The syntax for a url link is not the same in RST than in Markdown

>     conda create -n festim-env -c conda-forge fenics
>     source activate festim-env

Reviewer: Should we recommend using mamba for a faster installation?

```suggestion
    conda create -n festim-env
    source activate festim-env
    conda install mamba
    mamba install -c conda-forge fenics
```

Author: yeah for sure, it does take a while

FEniCS can be installed with Anaconda on MacOs and Linux. 
In order to use the Anaconda distribution on Windows, use the `Windows Subsystem for Linux <https://learn.microsoft.com/en-us/windows/wsl/install>`_. 
First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands::

    conda create -n festim-env

>     source activate festim-env

Reviewer: Shouldn't it be conda activate instead?

Author: I mean thats what I put initially, can change it back

In order to use the Anaconda distribution on Windows, use the `Windows Subsystem for Linux <https://learn.microsoft.com/en-us/windows/wsl/install>`_. 

> First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands 

Reviewer: ```suggestion
First `install Anaconda <https://docs.continuum.io/anaconda/install>`_ then run the following commands::
```

Just look at the other code blocks, you need to add :: to the paragraph before the code block

>     conda create -n festim-env <br/>
>     source activate festim-env <br/>
>     conda install mamba <br/>

Reviewer: ```suggestion
    conda create -n festim-env
    source activate festim-env
    conda install mamba
```
